<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(18, 89, 156);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>730</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Info Medico</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonElimina">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(18, 89, 156,0);
	color: rgb(170, 0, 0);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(15, 79, 139);
}
</string>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonModifica">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(18, 89, 156,0);
color: rgba(230, 235, 239,1);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgb(15, 79, 139);
}
</string>
   </property>
   <property name="text">
    <string>Modifica</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="LogoMedici">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>200</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../../MATERIALE/IconaMedico.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>290</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Dati personali del medico</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>290</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 89, 156);
border: 1px solid rgb(230, 235, 239);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 89, 156);
border: 1px solid rgb(230, 235, 239);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>290</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Credenziali del medico nel sistema</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 89, 156);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Data di nascita :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CodiceFiscale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Sesso">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="DataDiNascita">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>210</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: rgb(230, 235, 239);
QLineEdit {
background-color: rgb(18, 89, 156);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgb(230, 235, 239);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>Sheda_2</zorder>
  <zorder>Sheda</zorder>
  <zorder>LogoMedici</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>ButtonElimina</zorder>
  <zorder>ButtonModifica</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>Nome</zorder>
  <zorder>Cognome</zorder>
  <zorder>CodiceFiscale</zorder>
  <zorder>Sesso</zorder>
  <zorder>DataDiNascita</zorder>
  <zorder>Username</zorder>
  <zorder>Password</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
